scripts/select_msoas.py

```diff
@@ -37,6 +37,5 @@ def area(lad):
             f"No matches for LAD20NM = {lad} in data/raw_data/referencedata/lookUp.csv"
         )
 
-
```

Comment:
Why's there a change here? Running the black formatter puts this newline back

Reply:
No idea. Ignore it.